packages/daimo-api/src/network/bundlerClient.ts

```diff
@@ -85,6 +87,23 @@ export class BundlerClient {
         this.getOpHash(op, viemClient.publicClient),
         this.sendCompressedOpToBulk(compressed, viemClient),
       ]);
+
+      if (this.opIndexer) {
+        const opStart = Date.now();
+        const span = trace.getTracer("daimo-api").startSpan("bundler.submit");
+        this.opIndexer.addCallback(opHash, (userOp) => {
+          span.setAttributes({
+            opHash: userOp.hash,
+            txHash: userOp.transactionHash,
+            logIndex: userOp.logIndex,
```

Comment:
for consistency i'd make these, 'op.hash', 'op.tx_hash', 'op.log_index' as in router.ts:180